src/mono/System.Private.CoreLib/src/System/Math.Mono.cs

```diff
@@ -82,9 +82,6 @@ public partial class Math
         [MethodImpl(MethodImplOptions.InternalCall)]
         public static extern double FusedMultiplyAdd(double x, double y, double z);
 
-        [MethodImpl(MethodImplOptions.InternalCall)]
```

Comment:
can we clean these from sysmath.c, interp.c, mintops.def, icall-def-netcore.h and transform.c too?

Reply:
Absolutely, I've removed ILogB logic from those files. Please let me know if there is anything else I can modify.

Reply:
Great, thanks!
![image](https://user-images.githubusercontent.com/3840695/127028876-e03c47fb-d0b7-4a47-b46f-0546def014cb.png)

git stats are just perfect! 👌😅